docs/user-guide.md

```diff
@@ -109,18 +110,45 @@ Initial steps similar to those of the Single Instance installation.
 
    `./install-oracle.sh --help`
 
-1. Run installation. Only IP address of the first cluster node is required:
+1. Create cluster config file. The file `cluster_config.json` is provided as template
```

Comment:
Should we tell the user what directory they can find the template in?

Reply:
I think all other files described do not have any comments about their locations meaning they are in root directory. I would leave it without directory location